src/components/DataTable.tsx

```diff
@@ -354,9 +346,7 @@ export function DataTable<T extends object>(props: Props<T>) {
                       {props.customActions &&
                         props.customActions?.map(
                           (action: any, index: number) => (
-                            <React.Fragment key={index}>
-                              {action(resource)}
-                            </React.Fragment>
+                            <div key={index}>{action(resource)}</div>
```

Comment:
The reason for changing React.Fragment to a div HTML element is that React.Fragment doesn't have any events like onClick, but we really need it here. We need to close the dropdown after clicking on a particular option. In our case, if we don't do this, the modal and dropdown will open at the same time, which is very bad behavior IMO.